fs/layer/node.go

```diff
@@ -244,6 +244,26 @@ func (fs *fs) inodeOfStatFile() uint64 {
 	return (uint64(fs.baseInode) << 32) | 2 // reserved
 }
 
+// basicTelemetry wraps informational FUSE telemetry operations.
+func (fs *fs) basicTelemetry(ctx context.Context, operationName string, path string) {
+	if fs.logFSOperations {
+		log.G(ctx).WithFields(logrus.Fields{
+			"operation": operationName,
+			"path":      path,
+		}).Debug("FUSE operation")
+	}
+	if fs.operationCounter != nil {
```

Comment:
Is there a case where this would not already be set? If not might be worth removing, as it could create some confusion.

Reply:
We create a FUSE operation counter for every image, so it is always set. I think having a `nil` here is a good preventative measure against unexpected runtime errors, if we ever accidentally change any behavior.